lib/dot/definitions.def

```diff
@@ -3,7 +3,7 @@
     var $lvl = it.level;
     var $dataLvl = it.dataLevel;
     var $schema = it.schema[$keyword];
-    var $schemaPath = it.schemaPath + '.' + $keyword;
```

Comment:
> i.e. you get a line in lib\custom.js that looks like:
> var $schemaPath = it.schemaPath + it.utils.getProperty($keyword);

If you mean the file lib\dotjs\custom.js then that's what it should be.
It's not a generated code of validation function, it's a code that generates code of validation function :)